projects/packages/jetpack-mu-wpcom/src/features/holiday-snow/class-holiday-snow.php

```diff
@@ -186,49 +282,106 @@ public static function update_option_api( $input, $unfiltered_input ) {
 	 * @return void
 	 */
 	public static function register_settings() {
-		register_setting(
-			'general',
-			self::HOLIDAY_SNOW_OPTION_NAME,
-			array(
-				'type'              => 'boolean',
-				'description'       => esc_attr__( 'Show falling snow on my site', 'jetpack-mu-wpcom' ),
-				'show_in_rest'      => true,
-				'default'           => false,
-				'sanitize_callback' => function ( $value ) {
-					return (bool) $value;
+		foreach ( self::$holiday_snow_config as $option_name => $option_config ) {
+			if ( $option_config['type'] === 'boolean' ) {
```

Comment:
Nitpick: we could use Yoda contidion here as well as above in other places.

```suggestion
			if ( 'boolean' === $option_config['type'] ) {
```

Reply:
👍 Done in 62cb10a724c027e75118d6047db58551e3772352